libparsec/crates/client/src/workspace/merge.rs

```diff
@@ -150,7 +150,19 @@ pub(super) fn merge_local_folder_manifest(
     prevent_sync_pattern: Option<&Regex>,
     local: &LocalFolderManifest,
     remote: FolderManifest,
+    force_apply_pattern: bool,
```

Comment:
```suggestion
    pattern_already_applied_to_local: bool,
```
I'd rather rename this option to make it more explicit about when it is needed

Reply:
```suggestion
    pattern_not_applied_to_local: bool,
```

Should be the inverse, the option should be set to true if the pattern is not fully applied.
Otherwise if should inverse it's use in the if WDYT?